### PR TITLE
grep: add -A -B -C flags

### PR DESCRIPTION
**Pull request by pawiromitchel**
if you want more context around the search, add those flags

- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by navarroaxel** [comment]
@pawiromitchel the tldr-lint check 1 example description line and 1 command example line, that's why your proposal can't pass the check

File: pages/common/grep.md [hunk 1]
```diff
@@ -23,6 +23,16 @@
...
 
+You can also use these 3 flags to add more context of your search
+
+`-A num` = print x num of lines after each match
+
+`-B num` = print x num of lines before each match
+
+`-C num` = print x num of lines before and after each match
+
+`grep -C 3 "{{search_pattern}}" {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/grep.md:34]: This example is the same than above just use a short argument version.

Comment (pawiromitchel) [reply pages/common/grep.md:34]: sorry if this is spamming your inbox 😅

Comment (navarroaxel) [reply pages/common/grep.md:34]: np. This is a very popular command and it's hard to cover its functionality in just 8 basic examples

**Comment by pawiromitchel** [comment]
Ahhh there can only be 8 examples, got it 💯

**Comment by pawiromitchel** [comment]
I can close this PR then, no need for it haha

**Pull request closed without merging**